docs/code-of-conduct.md

```diff
@@ -14,17 +14,17 @@ We value diverse opinions, but we value respectful behavior more.
 
 Respectful behavior includes:
 
-* Being considerate, kind, constructive, and helpful.
-* Not engaging in demeaning, discriminatory, harassing, hateful, sexualized, or
+- Being considerate, kind, constructive, and helpful.
+- Not engaging in demeaning, discriminatory, harassing, hateful, sexualized, or
   physically threatening behavior, speech, and imagery.
-* Not engaging in unwanted physical contact.
+- Not engaging in unwanted physical contact.
 
 Some Google open source projects [may adopt][] an explicit project code of
 conduct, which may have additional detailed expectations for participants. Most
 of those projects will use our [modified Contributor Covenant][].
 
 [may adopt]: https://opensource.google/docs/releasing/preparing/#conduct
-[modified Contributor Covenant]: https://opensource.google/docs/releasing/template/CODE_OF_CONDUCT/
+[modified contributor covenant]: https://opensource.google/docs/releasing/template/CODE_OF_CONDUCT/
```

Comment:
Do you know what rule caused the capitalization to be updated here?

These files are part of the standard Google OSS repo example, and I don't think we want to be making changes to them.

Reply:
Hmmm... not sure why that happened and doesn't happen now. Will revert.

Reply:
So it did change this, and it can't seem to be switched off. However this is just the link reference and [it is case insensitive](https://spec.commonmark.org/0.25/#example-505). So the actual link is rendered correctly with upper case [as shown here](https://github.com/GoogleChrome/web-vitals/blob/4b84d071710797997a14d1698a8c5317b9329912/docs/code-of-conduct.md).

Good to keep this change?

Or, alternatively, moving these to inline links rather than reference links is probably more easily understood by most people used to GitHub markdown syntax, and also passes prettier, but is another change to the file.

Or ignore this file, but that means adding a `.prettierignore` file.

WDYT?

Reply:
> However this is just the link reference and [it is case insensitive](https://spec.commonmark.org/0.25/#example-505). So the actual link is rendered correctly with upper case [as shown here](https://github.com/GoogleChrome/web-vitals/blob/4b84d071710797997a14d1698a8c5317b9329912/docs/code-of-conduct.md).

Ah, I see. I missed that it was just the reference. Yeah, this seems fine.